diff --git a/node_modules/@capacitor-firebase/analytics/ios/Plugin/FirebaseAnalytics.swift b/node_modules/@capacitor-firebase/analytics/ios/Plugin/FirebaseAnalytics.swift
index 936d2c1..5f92a8a 100644
--- a/node_modules/@capacitor-firebase/analytics/ios/Plugin/FirebaseAnalytics.swift
+++ b/node_modules/@capacitor-firebase/analytics/ios/Plugin/FirebaseAnalytics.swift
@@ -8,6 +8,9 @@ import FirebaseAnalytics
 
     override init() {
         if FirebaseApp.app() == nil {
+            guard Bundle.main.path(forResource: "GoogleService-Info", ofType: "plist") != nil else {
+                return
+            }
             FirebaseApp.configure()
         }
     }
